src/frontend/src/routes/(new-styling)/manage/(authenticated)/security/+page.svelte

```diff
@@ -0,0 +1,206 @@
+    if (isNullish(googleClientId)) throw new Error("Missing Google client ID");
+    const { nonce, salt } = await createAnonymousNonce(
+      $authenticatedStore.identity.getPrincipal(),
+    );
+    const jwt = await requestJWT(createGoogleRequestConfig(googleClientId), {
+      mediation: "required",
+      nonce,
+    });
+
+    const { iss, sub, aud, name, email } = decodeJWTWithNameAndEmail(jwt);
+
+    if (
+      identityInfo.openIdCredentials.find((c) => c.iss === iss && c.sub === sub)
+    ) {
+      throw new Error("Account already linked");
+    }
+
+    const googleAddPromise = $authenticatedStore.actor.openid_credential_add(
+      $authenticatedStore.identityNumber,
+      jwt,
+      salt,
+    );
+    // Optimistically show as added
+    identityInfo.openIdCredentials.push({
+      aud,
+      iss,
+      sub,
+      metadata: [
+        ["name", { String: name }],
+        ["email", { String: email }],
+      ],
+      last_usage_timestamp: [],
+    });
+    displayAddCredentialDialog = false;
+
+    const googleAddResult = await googleAddPromise;
+
+    if ("Ok" in googleAddResult) {
+      identityInfo.fetch();
+    } else {
+      identityInfo.openIdCredentials = identityInfo.openIdCredentials.filter(
+        (cred) => !(cred.iss === iss && cred.sub === sub),
+      );
+      throw new Error(Object.keys(await googleAddResult.Err)[0]);
+    }
+  };
+
+  const handleRemoveGoogle = async (credential: OpenIdCredential) => {
+    const googleRemovePromise =
+      $authenticatedStore.actor.openid_credential_remove(
+        $authenticatedStore.identityNumber,
+        [credential.iss, credential.sub],
+      );
+    // Optimistically show as removed
+    identityInfo.openIdCredentials = identityInfo.openIdCredentials.filter(
+      (cred) => !(cred.iss === credential.iss && cred.sub === credential.sub),
+    );
+    removableOpenIdCredential = null;
+
+    const googleRemoveResult = await googleRemovePromise;
+
+    if ("Ok" in googleRemoveResult) {
+      identityInfo.fetch();
+    } else {
+      identityInfo.openIdCredentials.push(credential);
+      throw new Error(Object.keys(googleRemoveResult.Err)[0]);
+    }
+  };
+</script>
+
+<div class="text-text-primary">
+  <h1 class="mb-4 text-4xl">Security</h1>
+  <h2 class="mb-12 text-lg">
+    Settings and recommendations to keep your identity secure
+  </h2>
+  <Panel>
+    <div class="flex flex-col justify-between gap-5 p-4 md:flex-row">
+      <div>
+        <h3 class="mb-2 text-lg font-semibold">Access methods</h3>
+        <h4>Manage your passkeys, security keys, and linked accounts.</h4>
+      </div>
+
+      <div>
+        <Button
+          onclick={() => {
+            displayAddCredentialDialog = true;
+          }}
+          class="bg-bg-brand-solid text-text-primary-inversed text-[] top-0 flex w-full items-center justify-center gap-1 rounded-sm px-3.5 py-2 font-semibold md:max-w-fit"
+          >Add <Plus size="1.25rem" /></Button
```

Comment:
Should we name it "Link Google Account" for now?

Reply:
True, that's too big.

But it might be confusing to have "Add" and that trigger the Google link.